kolibri/core/assets/src/views/app-body/index.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+
+  <div class="body" :style="contentStyle">
+    <loading-spinner v-if="loading" class="align-to-parent" />
```

Comment:
`align-to-parent` class is not defined in this component.

Reply:
Ah right. I cleared it out because it didn't seem to be doing anything.